### DISCUSSION TITLE
How to import perspective camera from gltf file.  We have camera setup in gltf file. How can we import camera from gltf file. Is there any options ?

### DISCUSSION THREAD

**Discussion by manim86** [jnsmalm/pixi3d#187] [General]
How to perspective camera from gltf file.  We have camera setup in gltf file. How can we import camera from gltf file. Is there any options ? When i explore the data i see one 3d container with default camera name.

**Comment by jnsmalm** [comment]
You can find the entire contents of glTF file like this (it's in the "descriptor property" of the glTF asset object):

```
app.loader.load((_, resources) => {
  let gltf = resources["assets/teapot/teapot.gltf"].gltf.descriptor
  // Find camera somewhere in side the gltf file
  // Camera.main.position.set(?,?,?)
})
```
There is no automatic way to set the camera properties to what's available inside the glTF file so you have to do that manually something like the code above.
